modules/ui/src/app/pages/settings/settings.store.spec.ts

```diff
@@ -131,8 +133,25 @@ describe('SettingsStore', () => {
           interfaces: {},
           deviceOptions: {},
           internetOptions: {},
-          logLevelOptions: {},
-          monitoringPeriodOptions: {},
+          logLevelOptions: {
```

Comment:
I think we can reuse this static date from already defined const in store: https://github.com/google/testrun/pull/264/files#diff-e9bce944b1d60e7a174327c172cd491a19f7b82d9e35e9ce838a2f076ca595e7R50